### DISCUSSION TITLE
New

### DISCUSSION THREAD

**Discussion by Waqas843** [community/community#44313] [New to GitHub] [answered]
Want to Learn

**Comment by ghost** [comment]
same here waqas843 I to want to learn a lot

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to GitHub and the GitHub Community @Waqas843  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
